tICA/tICAPipeline.sh

```diff
@@ -608,12 +646,19 @@ do
             par_runjobs "$parLimit"
             ;;
         (ComputeTICAFeatures)
+            # now we have the dimensionality, set the sICA and tICA output strings
+            sICAActualDim=$(cat "$sICAoutfolder/most_recent_dim.txt")
```

Comment:
Is it correct to ignore the provided `--sicadim-override` argument if someone starts the pipeline from this step?  If in doubt, the logic should probably match the other places that check this.